source/table_view.tsx

```diff
@@ -32,12 +41,18 @@ export class TableView extends React.Component<Properties> implements
 
   constructor(props: Properties) {
     super(props);
+    this.state = {
+      rowHeight: 1,
+      rowsToShow: 1,
+      topRow: 0
+    };
     this.headerRefs = [];
     for(let i = 0; i < this.props.labels.length; ++i) {
       this.headerRefs[i] = null;
     }
     this.table = new SortedTableModel(this.props.model);
     this.table.connect(this.forceUpdate.bind(this, null));
```

Comment:
Okay good initial implementation but unfortunately this will trigger a render every time the table changes which is too slow. Only trigger a render when a visible change takes place. I leave it to you to define exactly what the definition of a visible change is.

Reply:
Should I cancel/close the pull request for now?